CHANGELOG.rst

```diff
@@ -14,6 +19,7 @@ A minor release adding some features and bug-fixes to the renku components. The
 - UI: correct bugs affecting the project's dataset page
 - renku-python: add Plan and project status support in Renku Python API
 - renku-python: add a custom git merge tool for merging renku metadata
+- renku-notebooks: switch git proxy sidecar to golang.
```

Comment:
I think you meant "gondola" 😆 

https://github.com/SwissDataScienceCenter/renku/runs/6665603908?check_suite_focus=true#step:5:135